### ISSUE TITLE
Update game-running scripts to use ES6 standards

### ISSUE THREAD

**Issue by Asuza** [JSJitsu/ai-battle-website#54]
Update code in the _gamerunner_ directory to more modern standards:
- Rather than using prototypes, use classes.
- Remove `var` statements in favor of `let` or `const`, and break up multi-var statements into separate lines. For example, `var a = 1, b = 2;` should become `let a = 1; let b = 2;` (on two lines). If the variable has no value, move the `let` where it makes sense. For example,
    ```
    var c = 3,
        d;

    d = 19;
    ```
    should be
    ```
    let c = 3;

    let d = 19;
    ```
- Use arrow functions (where it makes sense to) in place of standard functions.

**Comment by t3chguy** [comment]
though can't open the PR until 1st October :P

**Comment by Asuza** [comment]
No problem, @t3chguy. I'm just trying to get issues ready for the big day.

**Comment by t3chguy** [comment]
I presume to not use es6 imports as there appears to be no babel in place

**Comment by Asuza** [comment]
Anything that will run on Node 6 is fair game. None of that code runs in a browser, so nothing to worry about babel-wise.

**Comment by t3chguy** [comment]
Staged here https://github.com/JSJitsu/ai-battle-website/compare/master...t3chguy:master

(Will open PR in October)

**Comment by Asuza** [comment]
Thanks @t3chguy. In regards to https://github.com/JSJitsu/ai-battle-website/compare/master...t3chguy:master#diff-8535c90dfcec366ed93cc47755880f1aR16 I believe that can be removed. I can't find it used anywhere.

**Comment by Asuza** [comment]
Fixed via #58. Much appreciated, @t3chguy.
